### PR TITLE
Releases/v0.4.0

### PR DESCRIPTION
**Pull request by andrewjl-mux**
# API Changes

* Add: UploadOptions struct to contain all available MuxUpload options (#48)
* Add: Options to request or to skip input standardization (#48)
* Add: MuxUpload initializer APIs that accept AVAsset or PHAsset (#48)
* Add: MuxUpload.InputStatus enum to represent the current state of the upload and change handler (#48)

## New

* feat: Add support for input standardization, create a playable asset from your direct upload faster. Non-standard files go through an extra normalization step when they are ingested post-upload. When input standardization is requested from the SDK, input video is converted to a standard range of values on a best-effort basis (#48)

## Fixes

* Prevent integer overflow when calculating chunk request content ranges (#38)
* Prevent crash from chunk worker force unwrap (#47)
* Remove public methods from internal SDK classes (#51)
* Prevent removal of result handler properties when passing MuxUpload via UploadManager (#61)

Co-authored-by: Emily Dixon <edixon@mux.com>
Co-authored-by: AJ Lauer Barinov <102617203+andrewjl-mux@users.noreply.github.com>
Co-authored-by: AJ Lauer Barinov <abarinov@mux.com>
Co-authored-by: GitHub <noreply@github.com>

### CODE REVIEW

File: Tests/MuxUploadSDKTests/Upload Tests/ReporterTests.swift [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            appVersion: "3.2.1",
+            deviceModel: "iPad",
+            errorDescription: "foo",
+            inputDuration: 3.14,
+            inputSize: 1_500_000,
+            maximumResolution: "default",
+            nonStandardInputReasons: [],
+            platformName: "iPadOS",
+            platformVersion: "15.0.0",
+            sdkVersion: "0.4.1",
```
Comment (daytime-em) [inline Tests/MuxUploadSDKTests/Upload Tests/ReporterTests.swift:36]: Right?
```suggestion
            sdkVersion: "0.4.0",
```

**Review by daytime-em — CHANGES_REQUESTED** [review]
Everything builds and passes, I just wanted to make sure the version number was correct

File: Tests/MuxUploadSDKTests/Upload Tests/ReporterTests.swift (hunk 1 above)
Comment (andrewjl-mux) [reply Tests/MuxUploadSDKTests/Upload Tests/ReporterTests.swift:36]: This is test code, the version used here is immaterial

**Review by daytime-em — APPROVED** [review]
I have been informed that was test data

**Pull request merged**